test/fixtures/signs.yml

```diff
@@ -19,3 +19,9 @@ two:
   emergency: MyString
   emergency_detail: MyString
   visibility: 1
+
+private:
+  name: A Private Sign
+  template: default
+  location: Build Team Area
+  visibility: 1               # Yes, 1 means hidden/private.
```

Comment:
I agree. I am not at all happy about this but we missed it when it came in. Fixing it now would mean a datafix in production, which is something I don't want to hassle at the moment. So hopefully there are enough warning signs in the code that it won't be a problem. 

None of this confusion is visible to the user.